modules/post/windows/manage/wdigest_caching.rb

```diff
@@ -0,0 +1,101 @@
+require 'msf/core'
+require 'rex'
+require 'msf/core/auxiliary/report'
+
+class Metasploit3 < Msf::Post
+  include Msf::Post::Windows::Priv
+  include Msf::Post::Windows::Registry
+  include Msf::Post::Windows::Accounts
+  include Msf::Auxiliary::Report
```

Comment:
Out of all of these includes, it looks like only `Msf::Post::Windows::Registry` is required. Is there something that I'm missing? I don't see any references to functions provided by the other mixins.